packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -360,13 +368,13 @@ class _HeroState extends State<Hero> {
       'A Hero widget cannot be the descendant of another Hero widget.'
     );
 
-    final bool isHeroInFlight = _placeholderSize != null;
+    final bool placeheld = _placeholderSize != null;
```

Comment:
Why change the name here? 'placeheld' seems a bit more ambiguous to understanding that the hero is currently flying.

Reply:
This pr make it so after hero transition finishes, the hero widget in the original route will still be hidden. The placeheld (or isHeroInFlight previously) is controlling whether to hide the hero. After this change, we want to hide hero even if it is not in flight. I feel the name become confusing, and that is why i renamed it. I am open to any name suggestion tho

Reply:
Maybe a better name would be: showPlaceholder?